addons/payment/static/src/js/express_checkout_form.js

```diff
@@ -0,0 +1,138 @@
+odoo.define('payment.express_checkout_mixin', require => {
```

Comment:
For new js code please use ES6 like module declaration
```suggestion
/** @odoo-module **/
```